default/specials.txt

```diff
@@ -290,7 +290,7 @@ Special
             scope = Source
             activation = OwnerHasTech name = "LRN_XENOARCH"
             effects = [
-                AddSpecial name = "ANCIENT_RUINS_DEPLETED_SPECIAL"
+                AddSpecial name = OneOf("ANCIENT_RUINS_DEPLETED_SPECIAL", "ANCIENT_RUINS_DEPLETED_BANFORO_SPECIAL", "ANCIENT_RUINS_DEPLETED_KILANDOW_SPECIAL", "ANCIENT_RUINS_DEPLETED_MISIORLA_SPECIAL")
```

Comment:
OK, so the plan is to have a 1 in 4 chance of nothing and a 3 in 4 chance of one of the new guys regardless of other results?